### ISSUE TITLE
Code feedback

### ISSUE THREAD

**Issue by indefinit** [debbilitt/homework2#1]
https://github.com/debbilitt/homework2/blob/52ec9bcb957eec9bed1ca9e1559a6765175a368d/finalHW2/src/ofApp.cpp#L209

Overall, I'm really impressed by your work on this project. You seemed to present a clear goal for the project and when some items didn't work exactly as you would have liked, you came up with a good workaround. 

Now, specifically about the technique of blob contour tracking, you were so close! Think about using the blobs from the contour finder and either doing a distance calculation from the perimeter of your blob, or each individual point along the contour. The latter option would be more process intensive, but would yield higher fidelity results than doing a perimeter check.

If you go with the hi fi option:
the first function to use when checking distance across a 2-dimensional space is the distance() or squareDistance() method, which is a member of the ofVec2f class. Documentation here: http://openframeworks.cc/documentation/math/ofVec2f/#!show_squareDistance

but before you can check distance between two ofPoints or ofVec2f (remember these are the same thing behind the scenes), you need to loop through your blobs and pull out the contours.
quick code meant for general example:
```
contourFinder.findContours();
for(auto &blob : contourFinder.blobs){
  for(auto &pt : blob.pts){
    //pt.squareDistance(centerPointOfText)
 }
}
```
If you go with the quick but faster option:
There is an even simpler way to check whether your text has intersected with your blob, and the function you'd want to use is the ofRectangle::intersects. Documentation here: http://openframeworks.cc/documentation/types/ofRectangle/#!show_intersects
Basically you need to find the bounding rectangle of your text, as well as the bounding rectangle of your blob.

Fortunately, ofxCvBlob has a method that will return an ofRectangle, to be used exactly for this purpose: http://openframeworks.cc/documentation/ofxOpenCv/ofxCvBlob/#show_boundingRect

Let me know if you have any other questions about these methods over Slack. Nice work on this!
